tests/setupJellyfish.ts

```diff
@@ -1,5 +1,5 @@
 import { Wait, DockerComposeEnvironment } from "testcontainers";
-import { setupState } from "./globalSetupState";
+import { setupState } from "./globalSetupState.js";
```

Comment:
I think, you could remove file extension
```suggestion
import { setupState } from "./globalSetupState";
```